### DISCUSSION TITLE
issue running video.py example in windows .

### DISCUSSION THREAD

**Discussion by AlanEbell** [gopro/OpenGoPro#142] [Q&A] [answered]
C:\Users\alane\PycharmProjects\GoProMesh\venv\Scripts\python.exe C:/Users/alane/PycharmProjects/GoProMesh/sdk_wireless_camera_control/open_gopro/demos/video.py
usage: video.py [-h] [-i IDENTIFIER] [-l LOG] [-o OUTPUT] [-w WIFI_INTERFACE]
                record_time
video.py: error: the following arguments are required: record_time

Process finished with exit code 2

I saw another discussion regarding this issue and I am having a heck of a time resolving it.  I am using Pycharm and working in a python 3.8 virtual environment.  Interestingly the Photo.py demo works just fine, but for the life of me I can seem to figure this out.

I am not a full time programmer so github and some of this stuff is a bit confusing to me.  Thanks in advance for any ideas that I can try to solve this.

**Comment by tcamise-gpsw** [comment, marked as answer]
Hello. You're just missing the record_time argument. For example, to record with 2 seconds run:

```
gopro-video 2
```
